media-sound/twolame/twolame-0.4.0.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A MPEG Audio Layer 2 encoder"
+DESCRIPTION="TwoLAME is an optimised MPEG Audio Layer 2 (MP2) encoder based \
+on tooLAME by Mike Cheng, which in turn is based upon the ISO dist10 code and \
+portions of LAME."
+HOMEPAGE="https://www.twolame.org"
+COPYRIGHT="2004-2018 The TwoLAME Project"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/twolame/twolame-$portVersion.tar.gz"
+CHECKSUM_SHA256="cc35424f6019a88c6f52570b63e1baf50f62963a3eac52a03a800bb070d7c87d"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	twolame$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libtwolame$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
```

Comment:
Looking at configure checks, libFLAC, libogg and libvorbis are not used (neither in REQUIRES nor REQUIRES_devel), (no mention from pkg-config also), ports like Only libsndfile should be needed (checked some recipes at repology for this)